style_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
@jsolbrig @cpcamacho I think we want to remove this script ?  Replaced by https://github.com/NRLMMD-GEOIPS/geoips/pull/78/files#diff-e8f6240a257a7aa0b892c9bbe434511f06f7fee30d1ac6bbb712c6fe65b53fe9

Reply:
Yes, this should be removed.